leverage/modules/project.py

```diff
@@ -279,6 +280,16 @@ def load_project_config():
         raise Exit(1)
 
 
+def validate_config(config: dict):
+    """
+    Run a battery of validations over the config file (project.yaml).
+    """
+    if not re.match(r"^[a-z0-9]([a-z0-9]|-){1,23}[a-z0-9]$", config["project_name"]):
+        raise ExitError(1, "Project name is not valid. Only lower case alphanumeric characters are allowed.")
```

Comment:
```suggestion
        raise ExitError(1, "Project name is not valid. Only lowercase alphanumeric characters are allowed.")
```
It also seems that `lowercase` altogether is the more accepted spelling: https://english.stackexchange.com/questions/59409/lowercase-lower-case-or-lower-case#59413
Sorry for making it in a different suggestion I thought it was alright but it kept bugging me 😅